crates/bevy_solari/src/realtime/gbuffer_utils.wgsl

```diff
@@ -22,7 +22,7 @@ fn gpixel_resolve(gpixel: vec4<u32>, depth: f32, pixel_id: vec2<u32>, view_size:
     let roughness = clamp(perceptual_roughness * perceptual_roughness, 0.001, 1.0);
     let props = unpack4x8unorm(gpixel.b);
     let reflectance = vec3(props.r);
-    let metallic = props.g;
+    let metallic = saturate(props.g); // TODO: Not sure why saturate is needed here to prevent NaNs
```

Comment:
Might be something wrong with our gbuffer encoding? @DGriffin91

Reply:
Seems like any bit pattern of gpixel.b should be valid in respect to metallic. It seems sus if unpack4x8unorm is returning something that's not 0.0..=1.0